SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorCommandLineArgsManager.h

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+#include "Templates/SharedPointer.h"
+
+class FReply;
+
+typedef TSharedRef<class ILauncher> ILauncherRef;
+typedef TSharedPtr<class ILauncherWorker> ILauncherWorkerPtr;
+typedef TSharedRef<class ILauncherProfile> ILauncherProfileRef;
+
+#if ENGINE_MAJOR_VERSION >= 4 && ENGINE_MINOR_VERSION >= 24
+#define ENABLE_LAUNCHER_DELEGATE
+#endif
+
+class FSpatialGDKEditorCommandLineArgsManager
```

Comment:
We already have `SpatialCommandUtils`. This class only takes care of pushing command line arguments to mobile devices, right?
How about we name it: `SpatialMobileCommandLineUtils` or `SpatialMobileCommandLineUtils`

Reply:
SpatialCommandUtils is a static class only interface. 
Utils always mean a series of static interface in one static class.
But FSpatialGDKEditorCommandLineArgsManager have a variable as a flag of device. it also need manage some callback function registered from launcher event.